model_train_iso_f.py

```diff
@@ -64,7 +64,7 @@ def prediction_iso_f(self):
         self.model_iso = self.model_from_to_pickle(is_writing=False)
         self.get_x_values(is_for_prediction=True)
         self.model_iso.n_jobs = -1
-        self.test[self.model_params['args']['pred_field']] = self.model_iso.predict(self.test[self.features].values)

```

Comment:
No need to recall the test data with features. The feature set has been already assigned to X variable